KsApi/mutations/inputs/PostCommentInput.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+public struct PostCommentInput: GraphMutationInput {
+  let body: String
+
+  /// This is base64 encoded ID of the Project-{projectId}}
```

Comment:
Do we have confirmation on how to handle this?

Reply:
Yeah @leighdouglas verified that on Android they have a reference to the graphID and can pass that in as the commentableId. I think the same thing should work for us.